tests/urlpatterns/tests.py

```diff
@@ -130,6 +130,13 @@ def test_invalid_converter(self):
         with self.assertRaisesMessage(ImproperlyConfigured, msg):
             path('foo/<nonexistent:var>/', empty_view)
 
+    def test_space_in_route(self):
+        with self.assertRaisesMessage(
+            ImproperlyConfigured,
+            "URL route 'space/<int: num>' contain invalid character space."
```

Comment:
Can you format this like the case above? (With a `msg` variable, and then on a single line?)